### ISSUE TITLE
Dataset features

### ISSUE THREAD

**Issue by Dayire** [zlwang-cs/TransModality#2]
Hey!

Great work done, keep going!

We may use your work as a basis for ours and for that I would need the features you used in the pickle files. Could you upload those?

Thanks!

**Comment by zlwang-cs** [comment]
Thank you for your comments.

As for the features, we used the features offered in project: https://github.com/soujanyaporia/multimodal-sentiment-analysis and project: https://github.com/soujanyaporia/contextual-multimodal-fusion. Please download features there.

**Comment by Dayire** [comment]
Thank you, got it,

Could you list the dependencies requirements? (Keras and Tensorflow version mostly!)

Thanks again!

**Comment by zlwang-cs** [comment]
Yes, I know the TF dependency is hard to handle but I lost the details about it. I just remember the TF version is 1.X.

**Comment by Dayire** [comment]
I believe I figured it out, I'm using TF 1.15.0 with keras 2.3.1

I'm trying to reproduce the results you mentioned in the paper (on MOSI and IEMOCAP) but I'm unable to do so.I still have some issues though, if you can help me:

1. In this part, you are using 6 loss functions
`
       loss=[keras.losses.mae,
              keras.losses.mae,
              keras.losses.mae,
              keras.losses.mae,
              keras.losses.categorical_crossentropy,
              keras.losses.mae],
        loss_weights=[10, 10, 10, 10, 1, 0.5],
`
Whereas the model output has only 5 outputs:
`
outputs=[vt_decode, at_decode, tv_decode, ta_decode, classify_score])
`
resulting in a Keras compilation error. I removed the last mae loss and its weight (0.5), it works but I cannot achieve your results..

Any solution? 

Thanks again!
